src/components/Search.js

```diff
@@ -93,12 +93,19 @@ export const Search = () => {
     }
   }, [consentGranted, isOnline])
 
+  const hideW = (searchResults || !isOnline || loading)
+  const showSearchBar = allowUsage()
+  const showResultsList = isOnline && searchResults && !loading
+  const showLoading = isOnline && loading
+  const showOfflinePanel = !isOnline
```

Comment:
Love it, was thinking of something similar. This will be useful with tips and feed features coming up